operational/aws/instance_scheduled_events/AWS_Instance_Scheduled_Events

```diff
@@ -0,0 +1,261 @@
+        "Filter.1.Value.1": instance_id
+      },    
+    }
+  EOS
+end
+
+#Process the response data, check for the tags and generate a list of scheduled EC2 instances
+script "js_ec2_filter_map", type: "javascript" do
+  parameters "ds_ec2_list_with_tags", "param_exclude_tags"
+  result "content"
+  code <<-EOS
+    var param_exclude_tags_lower=[];
+    for(var j=0; j < param_exclude_tags.length; j++){
+      param_exclude_tags_lower[j]=param_exclude_tags[j].toString().toLowerCase();
+    }
+
+    var content=[]
+    for(var i=0; i<ds_ec2_list_with_tags.length; i++){
+      ec2=ds_ec2_list_with_tags[i]
+
+      //Check, if the tag present in entered param_exclude_tags, ignore the EC2 instance if the tag matches/present.
+      var tags = ec2['tags']
+      var isTagMatched=false
+      var tagKeyValue=""
+      for(var j=0; j < tags.length; j++){
+        tag = tags[j]
+        if((param_exclude_tags_lower.indexOf((tag['tagKey']).toLowerCase()) !== -1) || (param_exclude_tags_lower.indexOf((tag['tagKey']+'='+tag['tagValue']).toLowerCase()) !== -1)){
+          isTagMatched = true;
+        }
+        // Constructing tags with comma separated to display in detail_template
+        if((tag['tagValue']).length > 0){
+          tagKeyValue = tagKeyValue + ', '+ tag['tagKey']+'='+tag['tagValue']
+        }else{
+          tagKeyValue = tagKeyValue + ', '+ tag['tagKey']
+        }
+      }
+
+      //If EC2 instance tag does not match with entered param_exclude_tags and then check if any of the scheduled event type exists  
+      var scheduled_event_type_array = ["instance-retirement","instance-reboot","instance-stop","system-reboot","system-maintenance"]
+      var event_set_type = ec2['scheduled_event_type']	  
+      if(!(isTagMatched)  && scheduled_event_type_array.indexOf(event_set_type) !== -1){
+        content.push({
+          instance_id: ec2['instance_id'],
+          region: ec2['region'],
+          availability_zone: ec2['availability_zone'],
+          instance_state: ec2['instance_state'],
+          tagKeyValue:(tagKeyValue.slice(2)),
+          scheduled_event_type: ec2['scheduled_event_type'],
+          scheduledevent_start_time: ec2['scheduledevent_start_time'],
+          scheduledevent_end_time: ec2['scheduledevent_end_time'],          
+        })
+      }
+    }
+  EOS
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "policy_ec2_scheduled_instance_list" do
+  validate $ds_ec2_scheduled_instance_map do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} Scheduled Events on EC2 Instances Found in AWS"
+    detail_template <<-EOS
+# List of Scheduled Events on EC2 Instances
+| Instance_Id| Region | Availability_Zone | Instance_State | Scheduled_Event_Type  | Scheduled_Event_StartTime | Scheduled_Event_EndTime |  TAG_Value |
```

Comment:
I haven't noticed this in other policies, but the column labels should not include the underscore _

Reply:
@cdwilhelm  
We have updated by removing underscore. Thanks